package.json

```diff
@@ -18,8 +18,9 @@
     "test:smoke": "npm-run-all -s test:smoke:test test:smoke:report",
     "test:smoke:test": "mkdir -p build/reports && cross-env TESTS=smoke cucumber-js src/test/smoke/features/ --require 'src/test/common/steps/*.js' --require 'src/test/smoke/config' -f json:build/reports/smoke-report.json",
     "test:smoke:report": "node src/test/smoke/smoke-report.js",
-    "test:compatibility": "mkdir -p build/reports && cross-env TESTS=compatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
-    "test:compatibility:debug": "node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
+    "test:compatibility": "mkdir -p build/reports && cross-env TESTS=compatibility grunt compatibility",
+    "test:compatibility:debug": "mkdir -p build/reports && cross-env TESTS=compatibility grunt compatibility",
```

Comment:
These commands look identical

Reply:
We can probably drop the debug test

Reply:
@YetAnotherMatt We've removed test:compatibility:debug as this didn't work on master for myself or Mike